doc/users/cheatsheets/matplotlib_for_matlab_users.rst

```diff
@@ -0,0 +1,71 @@
+.. _matplotlib-for-matlab-users:
+
+================================
+WIP: Matplotlib for Matlab users
+================================
+
+Introduction
+============
+
+If it is not obvious enough from its name, the Matplotlib was originally
+developped as a migration and improvement of the namesake MATLAB® plotting
+utilies in Python. Throughout its :doc:`/users/history`, lots
+of effort was made to  strike a resembling interface to MATLAB®. After more
+than a decade of development, Matplotlib has become a major part of the Python
+ecosystem. It grows far beyond its original purpose of a MATLAB®
+plotting emulater. This page is intended to be a reference for new matplotlib
+users who are coming from MATLAB®. It collects common mistakes and useful
+wisdoms to help the transition.
```

Comment:
The intent here is great, but the language is a bit too informal for this type of document. In particular, please try to avoid language like "If it is not obvious" in teaching docs since that may make people feel bad. Also, while parts of Matplotlib where inspired by matlab, it was not trying to be a MATLAB clone and we avoid language that says that for legal reasons. I think it's fine to just have the following:

```This page is intended to be a reference for new matplotlib users who are coming from MATLAB®```

Reply:
Points taken. I got carried away :p